lib/buffer.js

```diff
@@ -1232,12 +1232,14 @@ function btoa(input) {
   return buf.toString('base64');
 }
 
-// Refs: https://infra.spec.whatwg.org/#forgiving-base64-decode
-const kForgivingBase64AllowedChars = [
+const asciiWhitespaceCharacters = [
```

Comment:
I moved this out of `kForgivingBase64AllowedChars`

Reply:
```suggestion
const kForgivingBase64AllowedChars = [
```